packages/neon/neon/lib/src/widgets/error.dart

```diff
@@ -21,12 +22,24 @@ class NeonError extends StatelessWidget {
     super.key,
   });
 
+  /// The error object.
+  ///
+  /// Can be of type [String] or [Exception], various subtypes of `Exception` are also handled separately.
   final dynamic error;
```

Comment:
What happens if this is null?

Either add it to the docs or make it non nullable (by making it an `Object`)

Reply:
I thought about this as well, it should probably be non-nullable

Reply:
Ah it has to be nullable so we can easily pass in `Result.error`, but using `Object?` still makes it better